clean-abap/CleanABAP.md

```diff
@@ -4606,6 +4606,15 @@ by redirecting them to test data defined in the unit test without interfering wi
 
 - Use the CDS test environment (`CL_CDS_TEST_ENVIRONMENT`) to test your CDS views.
 
+- Use the AUTHORITY-CHECK test environment (`CL_AUNIT_AUTHORITY_CHECK`) to withdraw authorizations during the runtime of the test.
+  This helps to test behavior of the code for different authorizations.
+
+```ABAP
+    DATA(authority_check_controller) = cl_aunit_authority_check=>get_controller( ).
+    DATA(authority_object_set) = cl_aunit_authority_check=>create_auth_object_set( user_role_authorizations ).
```

Comment:
I think an example of what the user_role_authorizations look like would be nice as well...

Reply:
@sautermi0  I will add this in a future request. As of now I would like to complete the current one.